Modules/Loadable/Markups/MRML/vtkMRMLMarkupsDisplayNode.h

```diff
@@ -66,26 +70,44 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsDisplayNode : public
                                    unsigned long /*event*/,
                                    void * /*callData*/ ) override;
 
-  /// Convenienve function for getting the displayable markups node
+  /// Convenience function for getting the displayable markups node
   vtkMRMLMarkupsNode* GetMarkupsNode();
 
-  /// Active item (item that the mouse is hovered over).
-  /// This propoerty is computed on-the-fly and saved to file.
-  vtkGetMacro(ActiveComponentType, int);
+  /// Active item (item that the mouse or other interaction context is hovered over).
```

Comment:
```
  /// Get active component type
  /// If any, return the activated widget component type for the given context. If no context is provided, "mouse" context is assumed.
  /// A widget component is activated in given context after it is "hovered over".
  /// This property is computed on-the-fly and saved to file.
  int GetActiveComponentType(std::string context="");
```

Reply:
It may be sensible to clarify what the `and saved to file` means

Reply:
component is used in the API, but item is sometimes used in the documentation. I tried to consistently use `component`

Reply:
I'll fix the mentions of "item".

> It may be sensible to clarify what the and saved to file means
Actually I'm not sure what it means. Need to ask @lassoan

Reply:
https://github.com/Slicer/Slicer/commit/ae9ae617f00b04fe0c19334ce1ce3dcc4453f07d#r34289815